### DISCUSSION TITLE
TR2X Features

### DISCUSSION THREAD

**Discussion by Checkm8Croft** [LostArtefacts/TRX#1679] [Ideas]
I have some ideas for TR2X:

- Widescreen patch (because is already on 4:3 and not 16:9)
- PSX Bars
- Bar Scaling
- Change bar colors
- "-gold" command for TR2Gold
- Set fly mode command 
- Load previous console command with up arrow
- Add a console command for play and stop a song

**Comment by aredfan** [comment]
Hi @Checkm8Croft, these are all great ideas.  We also have issues/PRs for most of these suggestions.

-  Widescreen patch (because is already on 4:3 and not 16:9)
https://github.com/LostArtefacts/TRX/issues/1647
- PSX Bars
https://github.com/LostArtefacts/TRX/issues/1611
- Bar Scaling
https://github.com/LostArtefacts/TRX/pull/2782
- Change bar colors
https://github.com/LostArtefacts/TRX/issues/1637
- "-gold" command for TR2Gold
https://github.com/LostArtefacts/TRX/issues/1621
- Set fly mode command
https://github.com/LostArtefacts/TRX/issues/1642
- Load previous console command with up arrow
https://github.com/LostArtefacts/TRX/issues/1571
- Add a console command for play and stop a song
https://github.com/LostArtefacts/TRX/pull/2437
